apps/backend/package.json

```diff
@@ -26,6 +26,7 @@
     "debug": "^4.3.4",
     "dotenv-cli": "^7.4.2",
     "drizzle-orm": "^0.31.2",
+    "es-toolkit": "1.16.0-dev.477",
```

Comment:
Should we be using the `-dev.477` version? We probably just want `1.16.0`, right?

Reply:
I had a TS issue with the the `1.16.0`, the types not being up to date.
They claim to be the "Lodash replacement" but it's still a bit "bleeding edge" 😄